dev/io.openliberty.reporting/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops

```diff
@@ -0,0 +1,54 @@
+#*******************************************************************************
+# This file follows the WebSphere Message Guidelines.
+# For more information, visit:
+# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
+# 
+#-----------------------------------------------------------------------------------------------------------------------------
+# Message prefix block: CWWKF1700 - CWWKF1799
+#-----------------------------------------------------------------------------------------------------------------------------
+# Message prefix block for io.openliberty.reporting: CWWKF1700 - CWWKF1799
+
+CWWKF1700.reporting.is.enabled=CWWKF1700I: The checking for security vulnerabilities is enabled. Information about this Liberty runtime will be analyzed against known vulnerabilities.
+CWWKF1700.reporting.is.enabled.explanation= The checking for security vulnerabilities is currently enabled.
+CWWKF1700.reporting.is.enabled.useraction=To disable the checking for security vulnerabilities, specify <cveReporting enabled="false" /> in the server.xml file.
+
+CWWKF1701.reporting.is.disabled=CWWKF1701I: The checking for security vulnerabilities is disabled.
+CWWKF1701.reporting.is.disabled.explanation=The checking for security vulnerabilities is currently disabled.
+CWWKF1701.reporting.is.disabled.useraction=To enable the checking for security vulnerabilities, specify <cveReporting enabled="true" /> in the server.xml file. 
+
+more.information.message=For more information, you can review {0}
+
+CWWKF1702.cve.found=CWWKF1702W: Based on an assessment of the Liberty runtime, it is recommended that you review the following CVEs: {0}
+CWWKF1702.cve.found.explanation=Security vulnerabilities have been found.
+CWWKF1702.cve.found.useraction=Follow the message to resolve what has been found.
+
+CWWKF1703.cve.not.found=CWWKF1703I: Based on analysis of the Runtime, no security vulnerabilities requiring further analysis were found.
+CWWKF1703.cve.not.found.explanation=There is no security vulnerabilities found at this time.
+CWWKF1703.cve.not.found.useraction=The following message has no issues.
+
+CWWKF1704.incorrect.url=CWWKF1704W: The specified URL is not correct and a connection cannot be made: {0}
+CWWKF1704.incorrect.url.explanation=No connection can be made for the provided URL.
+CWWKF1704.incorrect.url.useraction=Verify that the URL is correct.
+
+CWWKF1705.failed.response=CWWKF1705W: Unable to retrieve relevant CVE information due to a communication issue with the CVE Reporting Service.
+CWWKF1705.failed.response.explanation=The service was not able to give a response.
+CWWKF1705.failed.response.useraction=the service was not able to provide a response.
+
+CWWKF1706.issue.parsing=CWWKF1706W: Issue when parsing Product Information.
+CWWKF1706.issue.parsing.explanation=There was an issue when parsing the Product Information.
+CWWKF1706.issue.parsing.useraction=The service cannot successfully parse the product information.
```

Comment:
The useraction "The service cannot ["did not"?] successfully parse the product information." would be better as the explanation.

Maybe use for the useraction "Fix the parsing issue and try again." or explain what the user needs to do.